orbit/pkg/packaging/windows.go

```diff
@@ -177,6 +217,20 @@ func BuildMSI(opt Options) (string, error) {
 	return filename, nil
 }
 
+func checkWine(wineChecked bool) error {
+	if !wineChecked && runtime.GOOS == "darwin" {
```

Comment:
similar question above, is the plan to require wix on amd64?

Reply:
Not required. But you can use `--local-wix-dir` to use it on amd64.